sycl/source/CMakeLists.txt

```diff
@@ -187,6 +185,18 @@ function(add_sycl_rt_library LIB_NAME LIB_OBJ_NAME)
     endif()
   endif()
 
+  # Link with LLVMSupport for shared utilities.
+  add_dependencies(${LIB_NAME} LLVMSupport)
+  target_link_libraries(${LIB_NAME} PRIVATE LLVMSupport)
+  target_include_directories(${LIB_OBJ_NAME} SYSTEM PRIVATE ${LLVM_MAIN_INCLUDE_DIR})
+  target_compile_definitions(${LIB_OBJ_NAME} PRIVATE LLVM_DISABLE_ABI_BREAKING_CHECKS_ENFORCING=1)
```

Comment:
What is this needed for? Should we add an explanatory comment here?

Reply:
I don't fully remember the reason it was needed, but it does not seem to be needed anymore, likely because it was coming from LLVMObject. When/if we get LLVMObject back in here we can reevaluate, but until then I'll just remove it.